.circleci/config.yml

```diff
@@ -25,31 +25,22 @@ workflows:
           context: dockerhub-deploy
           requires:
             - tools/setup
-      - tools/deploy-snapshot:
-          jarname: "hitchhiker-tree"
+      - tools/deploy:
           context:
-            - clojars-deploy
             - dockerhub-deploy
           filters:
             branches:
-              only: development
+              only: main
           requires:
-            - tools/setup
-            - tools/format
```

Comment:
Is there a reason why format is not being used here anymore?

Reply:
good catch